src/app/services/google-analytics/google-analytics.service.ts

```diff
@@ -1,9 +1,26 @@
 import { Injectable } from '@angular/core';
+import Analytics, { AnalyticsInstance } from 'analytics';
+import googleTagManager from '@analytics/google-tag-manager';
+import { environment } from '../../../environments/environment';
 
 @Injectable({ providedIn: 'root' })
 export class GoogleAnalyticsService {
-  public pageView(pageName: string, pageUrl: string) {
+  analyticsInstance: AnalyticsInstance;
+
+  constructor() {
+    this.analyticsInstance = Analytics({
+      app: 'cvs-app-vtm',
+      version: '0.0.1',
```

Comment:
Are we able to grab these from the `package.json` via the `name` and `version` properties?